CMakeLists.txt

```diff
@@ -312,7 +312,10 @@ set(FMT_DEBUG_POSTFIX d CACHE STRING "Debug library postfix.")
 set_target_properties(fmt PROPERTIES
   VERSION ${FMT_VERSION} SOVERSION ${CPACK_PACKAGE_VERSION_MAJOR}
   PUBLIC_HEADER "${FMT_HEADERS}"
-  DEBUG_POSTFIX "${FMT_DEBUG_POSTFIX}")
+  DEBUG_POSTFIX "${FMT_DEBUG_POSTFIX}"
+  COMPILE_PDB_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}"
+  COMPILE_PDB_NAME "fmt"
+  COMPILE_PDB_NAME_DEBUG "fmt${FMT_DEBUG_POSTFIX}")
```

Comment:
Please add a short comment clarifying that these are for MSVC 2017 only. We'll remove this workaround in the future once the compiler version is no longer supported.

Reply:
Happy to do that, but I'm not sure this should be looked at as a temporary workaround. Not having it leaves you at the mercy of the defaults and I'm worried that in some future VS version the defaults could change again to who knows what.

Would you still like me to describe it as a temporary workaround, or as something that simply has to be set for Visual Studio?

Reply:
> Would you still like me to describe it as a temporary workaround, or as something that simply has to be set for Visual Studio?

Yes, please. We don't need to guard against theoretical future MSVC regressions.

Reply:
OK, done!